resources/gui/layouts/game/CraftingWindow.json

```diff
@@ -519,6 +519,96 @@
         "Font": "sourcesansproblack,10",
         "TextScale": 1.0
       },
+      "ProductChanceLabel": {
+        "Bounds": "216,82,107,16",
+        "Padding": "0,0,0,0",
+        "AlignmentEdgeDistances": "0,0,0,0",
+        "AlignmentTransform": "0,0",
+        "Margin": "0,0,0,0",
+        "RenderColor": "255,255,255,255",
+        "Alignments": "",
+        "DrawBackground": true,
+        "MinimumSize": "1,1",
+        "MaximumSize": "4096,4096",
+        "Disabled": false,
+        "Hidden": false,
+        "RestrictToParent": false,
+        "MouseInputEnabled": false,
```

Comment:
Does this need to be true?

Reply:
labels don't need to have mouse input enabled if the propose is to scroll within the container (if the container has the mouse input already enabled).